docs/content/Contribution guidelines/operations/packaging-and-versioning.md

```diff
@@ -56,8 +56,8 @@ Following are some recent tags and what they correspond to.
 
 ## Core container image
 
-The core container runs the kcp server and the KubeStellar core
-controllers and initializes the kcp workspaces. It is built from some
+The core container runs a space provider server and the KubeStellar core
+controllers and initializes the space provider's spaces. It is built from some
```

Comment:
I think it would be better to say more specifically that it is the spaces used by KubeStellar that are being initialized, or perhaps say that spaces are created and initialized for use by KubeStellar.

Reply:
I think this is a bit too specific, but added language to make it so